### ISSUE TITLE
Webkit Windows binaries

### ISSUE THREAD

**Issue by Boscop** [Lichtso/JSC-Standalone#1]
Are these the most up-to-date Windows binaries of libwebkit on the internet?

Or do you know where I can find more recent ones?

**Comment by Lichtso** [comment]
No, they this repo is definitely not "up to date" (last commit is 4 years ago).
Also this repo contains only the JS-engine of WebKit, not the entire WebKit project.

https://webkit.org/webkit-on-windows/
If you find the time to build it yourself, it would be nice if you could make a pull request with the compiled binaries (for 32bit and 64bit) and header files.

**Comment by Boscop** [comment]
Yeah, I saw that page but it made me not want to build it :/
It even depends on iTunes...

Is there really no place online where someone uploads libwebkit windows binaries?

**Comment by Lichtso** [comment]
Not that I know of, but you could post it here if you find something
